frontend/pages/weights/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { GetServerSideProps, InferGetServerSidePropsType } from "next"
+import Head from "next/head"
+import { Button } from "../../components/Button/Button"
+import { Headline } from "../../components/Headline/Headline"
+import { ItemPreview } from "../../components/Item/ItemPreview"
+
+// As long as we do not have a weight. Let's work with Todo
+type Todo = {
+    userId: number
+    id: number
+    title: string
+    completed: boolean
+}
+
+type WeightsListProps = {
+    items: Todo[],
+    currentPage: number,
+    limit: number
+}
+
+/** Base List for weights */
+export default function WeightsList({ items, currentPage, limit }: InferGetServerSidePropsType<typeof getServerSideProps>) {
+    const siteTitle = `Page ${currentPage} - Discover weights`
+
+    return (<>
+        {/* Meta Tags */}
+        <Head>
+            <title>{siteTitle}</title>
+        </Head>
+
+        <div className="container">
+            {/* Headline */}
+            <Headline level={3}>All weights</Headline>
+
+            {/* Weights (todos) */}
+            <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 gap-5">
+                {items.map((item) => <ItemPreview key={item.id} name={item.title} weight={item.completed ? "✓" : "X"} imageUrl="https://via.placeholder.com/96.png" id={item.id.toString()} />)}
+            </div>
+
+            {/* Pagination */}
+            <div className="flex justify-center mt-5 md:mt-10">
+                {currentPage > 0 && <Button to={`/weights?page=${currentPage - 1}&limit=${limit}`} className="mr-5" kind="tertiary">Previous</Button>}
+                <Button to={`/weights?page=${currentPage + 1}&limit=${limit}`} kind="tertiary">Next</Button>
+            </div>
+        </div>
+    </>
+    )
+}
+
+export const getServerSideProps: GetServerSideProps<WeightsListProps> = async (context) => {
+    const currentPage = parseInt(context.query?.page as string ?? "0")
```

Comment:
Current page sollte bei 1 anfangen und dann hochzählen. Es gibt keine Seite 0.

Reply:
@Zoe-Bot Bitte reviewen: Ist in https://github.com/world-wide-weights/wwweights/pull/44/commits/63fd66f509eae43c92ba40967a7adf1b89e69a2a


Funktioniert nun so:

- Wenn ein Custom Limit gesetzt ist wird allgemein `limit`-Query angehängt. Wenn es dem default wert entspricht wird es weggelassen. 
- Wenn wir auf Seite 2 zurück auf Seite 1 gehen wird page weggelassen, da redundant. <== denk an Canonical Link

```
Ich bin auf Page 2:
prev: /weights
next: /weights?page=3

Ich bin auf Page 3:
prev: /weights?page=2
next: /weights?page=4

Ich bin auf Page 2 mit limit = 10
prev: /weights?limit=10
next: /weights?page=3&limit=10

...
```